### PR TITLE
Create our initial Agent Loop Script

### PR DESCRIPTION
**Pull request by dvargas92495**
Going to start experimenting with an initial MVP for an actual employee. This problem is vast and is a bit hard to project out everything we'll want this to agent to do. So gonna just publish an initial script and start iterating on the box directly.

The create employee action should probably create a box and run the install script, but not focused on onboarding for now.

### CODE REVIEW

File: data/main.ts [hunk 1]
```diff
@@ -163,10 +163,16 @@ const setupInfrastructure = async (): Promise<void> => {
         }
       );
 
+      new S3Object(this, "main_agent_js", {
+        bucket: mainWebsite.bucket,
+        key: "scripts/agent.js",
+        source: "./data/scripts/agent.js",
```
Comment (dvargas92495) [inline data/main.ts:169]: any public downloadable scripts will live in `https://samepage.network/scripts/*`

File: data/scripts/agent.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+});
+
+const run = async () => {
+  const vellum = new VellumClient({
+    apiKey: process.env.VELLUM_API_KEY!,
+  });
+  const root = process.cwd();
+  const workspaceRoot = path.join(root, "workspace");
+  const nextInputLog = path.join(workspaceRoot, "next-inputs.json");
+  const killSwitch = path.join(workspaceRoot, "kill.log");
```
Comment (dvargas92495) [inline data/scripts/agent.ts:30]: Our emergency kill switch

File: data/scripts/agent.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    }
+    console.log(new Date().toISOString(), "Starting a new agent loop...");
+    return true;
+  };
+
+  while (checkLoop()) {
+    const input = zInput.parse(
+      JSON.parse(fs.readFileSync(nextInputLog).toString())
+    );
+    const response = await vellum.executeWorkflow({
```
Comment (dvargas92495) [inline data/scripts/agent.ts:49]: Gonna have as much logic live on vellum as possible. In future worlds, both the workflow runner logic and the natural language model should live on the box itself too. I'm gonna have one employee in Vellum's cloud and one employee in SamePage's cloud.

**Review by mdroidian — APPROVED** [review]
🔥

**Pull request merged**